data-migrations/migrations/20240502202804-add_original_data_id_fields_private.js

```diff
@@ -0,0 +1,29 @@
+export const up = async (db, client) => {
+  const PrivateCloudRequest = db.collection('PrivateCloudRequest');
+  const PrivateCloudRequestedProject = db.collection('PrivateCloudRequestedProject');
+
+  const privateCloudRequests = await PrivateCloudRequest.find({ originalDataId: { $exists: false } }).toArray();
+
+  for (const privateCloudRequest of privateCloudRequests) {
+    const licencePlateVal = privateCloudRequest.licencePlate;
+    const privateCloudRequestCreated = new Date(privateCloudRequest.created.toISOString());
+
+    const matchingPrivateCloudRequestedProjects = await PrivateCloudRequestedProject.find({
+      licencePlate: licencePlateVal,
+      created: { $lte: privateCloudRequestCreated },
```

Comment:
I don't think `created` field is reliable field to query onto.

Reply:
matchingPrivateCloudRequestedProjects will be in array with the same `licencePlate` and only `created` is different

Reply:
let's discuss this in another PR. this PR is too big.